randovania/data/json_data/prime1.txt

```diff
@@ -1234,19 +1209,19 @@ Asset id: 1423896872
   > Dock to Storage Cave
       Power Bomb and Grapple Beam and Space Jump Boots and Morph Ball
   > Dock to Security Cave
-      Grapple Beam and Space Jump Boots and Morph Ball
+      Grapple Beam and Space Jump Boots
 
 > Dock to Storage Cave; Heals? False
   * Plasma Door to Storage Cave/Dock to Phendrana's Edge
   > Dock to Upper Edge Tunnel
       Power Bomb and Morph Ball
 
 > Dock to Security Cave; Heals? False
-  * Normal Door to Security Cave/Dock to Phendrana's Edge
+  * Morph Ball Door to Security Cave/Dock to Phendrana's Edge
   > Dock to Lower Edge Tunnel
-      Morph Ball
+      Trivial
   > Dock to Upper Edge Tunnel
-      Morph Ball
+      Trivial
```

Comment:
Security Cave Node, remove connection to lower edge tunnel, readd morph to upper edge tunnel